data/json/snippets/newspapers.json

```diff
@@ -0,0 +1,430 @@
+      },
+      {
+        "id": "one_year_old_news_4",
+        "text": "TELEPORTATION?  An unidentified source working in a major university contacted multiple media outlets this week with a farfetched story that is nonetheless corroborated by third-party physicists as \"shockingly possible\".  This source, nicknamed \"Deep Space\", describes secret government experimentation into teleportation, dating back to before 2008.  \"When I read the tagline, I just scoffed,\" said Dr. Alice Feyn, a physics professor at MIT.  \"Then I read on.  This is strongly reminiscent of Dr. Takatoshi's lost work.  It feels bizarre to say it, but I think there's a grain of truth to all of this.\"  Dr. Amy Takatoshi was a quantum physicist who went missing, along with all records of her work, in July 2007."
+      },
+      {
+        "id": "one_year_old_news_5",
+        "text": "LASERS - NEXT BIG THING?: Top defense researchers remarked during demonstration of a prototype today: \"We are still working on it, the laser weapon lacks power, but has a technically almost unlimited range,\" Colonel Sylver, a spokesperson for the miltary said.  Laser weapons have been used for years in large-scale platforms, especially as anti-ordnance weapons; this marks the first field use of human-sized energy weapons.\n\n\"There's no real advantage over conventional weaponry at this point,\" said analyst Mark Coza in an interview.  \"I think they're rolling them out mainly to intimidate China, and in the hopes that field testing leads to further improvements.\""
+      },
+      {
+        "id": "one_year_old_news_6",
+        "text": "DRUG USE ON THE RISE: Recent statistic suggest drug use in New England has climbed more than 20% in the last two years.  \"People are scared for their jobs, their country, even their life… of course some will turn to drugs,\" said an expert."
+      },
+      {
+        "id": "one_year_old_news_7",
+        "text": "EDITORIAL: DRUG USE SPIKE DUE TO ECONOMY.  It's no secret that President Oswald's promises of a booming new tech economy have fallen flat on their face.  Whether you're a Democrat or a Republican, we should all be ready to march on Washington and let Oswald know how we feel about broken promises and unemployment.  Instead, people are drowning their sorrows in drugs, as usual, and nothing will change."
+      },
+      {
+        "id": "one_year_old_news_8",
+        "text": "EDITORIAL: SOCIAL MEDIA HAS GOTTEN OUT OF HAND, AND GONE OUT OF COUNTRY.  We can no longer trust what we read, and it's all because of social media.  Although our newspaper takes painstaking efforts to source our data, our colleagues have forgotten what it means to be a journalist, and widespread falsehoods are repeated even by reputable publications as fact.  What is the ultimate source of all this misinformation?  It's anyone's guess, but I think we need look no further than our largets international trade partner and rival, China."
```

Comment:
```suggestion
        "text": "EDITORIAL: SOCIAL MEDIA HAS GOTTEN OUT OF HAND, AND GONE OUT OF COUNTRY.  We can no longer trust what we read, and it's all because of social media.  Although our newspaper takes painstaking efforts to source our data, our colleagues have forgotten what it means to be a journalist, and widespread falsehoods are repeated even by reputable publications as fact.  What is the ultimate source of all this misinformation?  It's anyone's guess, but I think we need look no further than our largest international trade partner and rival, China."
```